lib/Runtime/Base/FunctionBody.h

```diff
@@ -2428,6 +2431,9 @@ namespace Js
         FieldWithBarrier(bool) m_hasBailoutInstrInJittedCode : 1; // Indicates whether function has bailout instructions. Valid only if hasDoneCodeGen is true
         FieldWithBarrier(bool) m_pendingLoopHeaderRelease : 1; // Indicates whether loop headers need to be released
         FieldWithBarrier(bool) hasExecutionDynamicProfileInfo : 1;
+#ifdef _M_X64_OR_ARM64
+        FieldWithBarrier(bool) m_tag : 1;                     // Used to tag the low bit to prevent possible GC false references
+#endif
```

Comment:
yeah, in later commit, I carefully added tag for every 4 bytes. and this won't be build break either, for x86 m_tag is defined in another .h file :)